docker-compose.yaml

```diff
@@ -48,22 +48,15 @@ services:
       - "6379:6379"
       - "8265:8265"
       - "10001:10001"
-    command: bash -c "ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --redis-password=yourpassword --block" # pragma: allowlist secret
+    # https://docs.ray.io/en/releases-2.30.0/cluster/cli.html#ray-start for more info about the command
+    command: bash -c "ray start --head --dashboard-port=8265 --port=6379 --dashboard-host=0.0.0.0 --ray-client-server-port 10001 --block" # pragma: allowlist secret
```

Comment:
Should we make these configurable (in some other PR)?

Reply:
Good catch, we should! @njbrake can you open an issue for this?

Reply:
Done! https://github.com/mozilla-ai/lumigator/issues/668